File: Python3/LongestCommonPrefix.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:

        if not strs or len(strs) == 0:
            return ""

        for i in range(len(strs[0])):
            compare = strs[0][i]
            for j in range(1,len(strs)):
                if i == len(strs[j]) or compare != strs[j][i]:
                    return strs[0][0:i]

        return strs[0]

File: Golang/LongestCommonPrefix.go
func longestCommonPrefix(strs []string) string {

    if strs == nil || len(strs) == 0 {
        return ""
    }

    for i:=0; i < len(strs[0]); i++ {
        compare := strs[0][i]
        for j:=1; j < len(strs); j++ {
            if i == len(strs[j]) || compare != strs[j][i] {
                return strs[0][0:i]
            }
        }
    }

    return strs[0]
    
}